app-modules/engagement/src/Actions/DeliverEngagements.php

```diff
@@ -17,10 +17,12 @@ class DeliverEngagements implements ShouldQueue
     use Queueable;
     use SerializesModels;
 
+    // TODO Add another indicator to the engagements table to represent an engagement being scheduled
+    // This will allow us to scope this query down further, and prevent any overlap with sync dispatches
```

Comment:
Let's just make sure we have a ticket for this TODO to ensure this gets worked on during Tech Debt burndown

Reply:
Yep! Ticket here: https://engage.canyongbs.com/workgroups/group/3/tasks/task/view/601/